extension/agenthealth/handler/useragent/useragent.go

```diff
@@ -81,6 +83,14 @@ func (ua *userAgent) SetComponents(otelCfg *otelcol.Config, telegrafCfg *telegra
 			// trim the adapter prefix from adapted Telegraf plugins
 			name := strings.TrimPrefix(receiver.Type().String(), adapter.TelegrafPrefix)
 			ua.inputs.Add(name)
+			if name == common.JmxKey {
+				cfg := otelCfg.Receivers[receiver].(*jmxreceiver.Config)
+				targetSystems := strings.Split(cfg.TargetSystem, ",")
+				for _, system := range targetSystems {
+					targetSystem := name + "-" + system
+					ua.inputs.Add(targetSystem)
```

Comment:
can you provide context on what a target system is? What kinds of health metrics are we generating?

Reply:
Target system is jvm, tomcat, kafka. 

This is to get jmx usage metrics. We want to see how many people use jvm, tomcat, kafka. The output will be jmx jxm-tomcat jmx-kafka